components/cd-article-list/cd-article-list.html

```diff
@@ -0,0 +1,57 @@
+<link rel=stylesheet href="../cd-base.css" type="text/css" media=screen>
+<link rel=stylesheet href="../cd-flow/cd-flow.css" type="text/css" media=screen>
+<link rel=stylesheet href="cd-article-list.css" type="text/css" media=screen>
+
+<div class="cd-layout-container">
+  <div class="cd-container">
+    <ul class="cd-article-list cd-article-list--icon [ cd-flow ]">
+      <li>
+        <a href="#">
+          <svg class="cd-icon cd-icon--arrow-right" aria-hidden="true" focusable="false">
```

Comment:
These should probably have a width and height as per https://github.com/UN-OCHA/common_design/blob/cd-226-design-system/common_design.libraries.yml#L70